### DISCUSSION TITLE
Any guide on how to integrate Elsa Worklfows V3 with Abp.Io or abp.io commercial?

### DISCUSSION THREAD

**Discussion by robson-paproski** [abpframework/abp#19519] [Q&A] [answered]
Need some help, There is any guide available to integrate the newest Elsa Workflows with Asp Net Zero, or since its blazor, directly with Abp framework/Abp Commercial? I couldn't find any that would work with the latest version.

**Comment by maliming** [comment, marked as answer]
hi

https://community.abp.io/posts/using-elsa-workflow-with-the-abp-framework-773siqi9

If you have any problem with this article, You can add your comments on it.

**Reply by garymedina** [reply]
This article is old. Elsa version V3 is a full rewrite.

Has anyone successfully integrated Elsa v3? How please?

**Reply by jacekmichalski** [reply]
@maliming Could you update the article to Elsa v3

**Reply by maliming** [reply]
> @maliming Could you update the article to Elsa v3

There is a newer version of this article, which covers Elsa 3 with ABP Framework. You can check it at https://abp.io/community/articles/using-elsa-3-workflow-with-abp-framework-usqk8afg

**Comment by islamal5tib** [comment]
🫡 https://github.com/sfmskywalker/abp-elsa-poc
